TheJumpCloud/jumpcloud-ADMU
# Add user agent tracker

## Issues
* [CUT-3820](https://jumpcloud.atlassian.net/browse/CUT-3820) - CUT-3820 Add User Agent Tracker

## What does this solve?
This enables CUT team to track ADMU usage
## Is there anything particularly tricky?
n/a
## How should this be tested?
1. Run ADMU app and check DataDog for user agent data:  `@http.useragent:JumpCloud.ADMU_Appication* `
<img width="654" alt="image" src="https://github.com/TheJumpCloud/jumpcloud-ADMU/assets/97972790/dcb17e42-d449-4274-86de-125529ff88e8">

2. Run ADMU module and check DataDog for user agent data: `@http.useragent:JumpCloud.ADMU_PowershellModule* `
<img width="526" alt="image" src="https://github.com/TheJumpCloud/jumpcloud-ADMU/assets/97972790/763c3ee9-8d72-4d62-96ab-ae1aca81a479">
## Screenshots

[CUT-3820]: https://jumpcloud.atlassian.net/browse/CUT-3820?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
## Screenshots

[CUT-3820]: https://jumpcloud.atlassian.net/browse/CUT-3820?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
